docs/guide/custom_policy.rst

```diff
@@ -13,9 +13,49 @@ and other type of input features (MlpPolicies).
   which handles bounds more correctly.
 
 
+SB3 Policy
+^^^^^^^^^^
 
-Custom Policy Architecture
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+SB3 networks are separated into two mains parts (see figure below):
+
+- a features extractor (usually shared between actor and critic when applicable, to save computation)
```

Comment:
Given this bullet point is two sentences long (so the next sentence starts upper case) I think it's cleaner to have this start a notional new sentence too.
```suggestion
- A features extractor (usually shared between actor and critic when applicable, to save computation)
```